src/main/java/eu/dissco/orchestration/backend/service/SourceSystemService.java

```diff
@@ -108,6 +108,33 @@ private static boolean isEquals(SourceSystem sourceSystem, SourceSystem currentS
             currentSourceSystem.getLtcCollectionManagementSystem());
   }
 
+  @PostConstruct
+  public void setup() throws ApiException {
```

Comment:
So the idea of putting this in a postConstruct is that because the orchestration shuts down every night and reboots every day, we'll catch any discrepancies that way?

Reply:
That would be just the test environment. Disrepancies wouldn't/shouldn't occur, as the orchestration backend is the only one who creates the crons. We could screw it up manually but we just shouldn't. No each time we update the translatorImage env property the pod will restart and first put all the crons on the new image.

Reply:
:+1: